docs/rgb_decomposition.md

```diff
@@ -0,0 +1,53 @@
+RGB Decomposition
+=================
+
+RGB products generated with HyP3 use both co-pol (VV or HH) and cross-pol (VH or HV) backscatter values to generate a color image from SAR data. This approach facilitates visual interpretation by decomposing the signals into surface scattering with some volume scattering (red band), volume scattering (green band), and surface scattering with very low volume scattering (blue band). 
+
+The dual-pol SAR dataset first undergoes radiometric terrain correction (RTC) to adjust for the distortions present in the images due to the side-looking geometry of SAR data acquisition. The RTC images are output in amplitude scale, and the RGB values are calculated using a combination of the VV and VH (or HH and HV) pixel values. The output RGB GeoTIFF is scaled so that the values for each band range from 1 to 255.
+
+### Decomposition Calculations
+
+Note that dual-pol image acquisition may provide either VV/VH returns or HH/HV returns, depending on the sensor mode. The same decomposition approach can be applied to either mode; refer to the appropriate equations below. 
+
+To calculate the RGB values, the data is first separated into surface (P<sub>r</sub>) and volume (P<sub>v</sub>) scattering components as follows:
+
+##### VV/VH
+
+P<sub>r</sub> = |S<sub>vv</sub>| <sup>2</sup> - 3 |S<sub>vh</sub>| <sup>2</sup>
+
+P<sub>v</sub> = 4 |S<sub>vh</sub>| <sup>2</sup>
+
+##### HH/HV
+
+P<sub>r</sub> = |S<sub>hh</sub>| <sup>2</sup> - 3 |S<sub>hv</sub>| <sup>2</sup>
+
+P<sub>v</sub> = 4 |S<sub>hv</sub>| <sup>2</sup>
+
+A spatial mask is generated using a threshold (*k*, typically between -22 and -25 dB) applied to the P<sub>v</sub> data so that pixels with values below the threshold are included in the blue channel mask (M<sub>B</sub>) and those above the threshold are included in the red band mask (M<sub>R</sub>).
```

Comment:
Should we just say we use -24 dB for the threshold by default?

Reply:
It would be good to add it in, but I think keeping the range of usual values could be helpful. In the RGB Decomp HyP3 process we give options for the threshold value to use, and if people are wanting to generate the images on their own, it's handy to have the range. I agree that we should document the default, though, especially if we're going to be limiting the RGB decomp as an option for the RTC process and not preserving the ability to choose the threshold.

Reply:
Yeah, useful range and default is probably best :-)